src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -2807,7 +2807,7 @@ def _get_storage_key(self, storage_account, resource_group):
     @StorageAccountPreparer(name_prefix='adlsgen2', length=16, location=location, key='storage-account')
     def test_managed_private_endpoints(self):
         self.kwargs.update({
-            'name': 'AzureDataLakeStoragePE',
+            'name': 'AzureDataLakeStoragePE1',
```

Comment:
Could you please tell the reason why we changed the name?

Reply:
I rerun failed once and try to fix it by change name. Lately I found that the definite file groupID need to update and name is irrelated. Just forgot to revert back...